### PR TITLE
Knowings-category bug fix - Mustafe

### PR DESCRIPTION
**Pull request by Muste8**
### Changes
Give a short summary of the changes made in this PR. 
-Located bug and fixed it
-Added notice to let user know if there are no student knowings

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206866992204663/1207176121882452/f
### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
-There is another bug when you click the button 'Add another' and you run into an error

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

Screenshot 2024-05-21 at 8 53 38 AM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/knowing_categories/index.html.erb [hunk 1]
```diff
@@ -2,13 +2,15 @@
   <thead>
     <tr>
       <th class="px-6 py-3 bg-gray-50 text-left text-sm text-gray-500 uppercase tracking-wider">Category</th>
-      <th class="px-6 py-3 bg-gray-50 text-left text-sm text-gray-500 uppercase tracking-wider">What I know about <%= @student_knowings.first.student.first_name %></th>
+      <th class="px-6 py-3 bg-gray-50 text-left text-sm text-gray-500 uppercase tracking-wider">What I know about <%= @student_knowings.first.student.first_name if @student_knowings.present? %></th>
```
Comment (edithemmings) [inline app/views/knowing_categories/index.html.erb:5]: We still want the student first name to appear even if there are no student_knowings

Comment (Muste8) [reply app/views/knowing_categories/index.html.erb]: Nice catch! Took care of it
Screenshot 2024-05-21 at 9 24 50 AM

**Review by edithemmings — APPROVED** [review]
Nice!! 🎉🎉

**Review by KNAthari — APPROVED** [review]

**Review by asmanoor17 — APPROVED** [review]
Works as expected, great work!

**Pull request merged**